### PR TITLE
feature/OC-196/standards-documentation

### PR DESCRIPTION
**Pull request by RalkeyOfficial**

### CODE REVIEW

File: pwa/src/templates/documentation/standardsDocumentation/StandardsDocumentationTemplate.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+import { useTranslation } from "react-i18next";
+
+export const StandardsDocumentationTemplate: React.FC = () => {
+  const { t } = useTranslation();
+
+  return (
+    <Container layoutClassName={styles.container}>
+      <section className={styles.section}>
+        <Heading1>Gebruikte standaarden</Heading1>
+        <LeadParagraph>De standaarden die gebruikt worden door Open Catalogi</LeadParagraph>
```
Comment (lencodes) [inline pwa/src/templates/documentation/standardsDocumentation/StandardsDocumentationTemplate.tsx:29]: I suggest making this a bit larger of a text ~~and to link to https://forumstandaardisatie.nl/ within that text~~.

Maybe add a button underneath to the [forumstandaardisatie](https://forumstandaardisatie.nl/) _and_ a button linking to https://forumstandaardisatie.nl/open-standaarden/verplicht 

image

Comment (lencodes) [reply pwa/src/templates/documentation/standardsDocumentation/StandardsDocumentationTemplate.tsx:29]: image

File: pwa/src/templates/documentation/standardsDocumentation/StandardsDocumentationTemplate.module.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
+.container {
+  max-width: 1440px;
```
[the remark this answers is not in the document]
Comment (lencodes) [reply pwa/src/templates/documentation/standardsDocumentation/StandardsDocumentationTemplate.module.css:2]: Above comment is still relevant for future expansions on the container element, however, I'd suggest this container to be the default width.

[the remark this answers is not in the document]
Comment (lencodes) [reply pwa/src/templates/documentation/standardsDocumentation/StandardsDocumentationTemplate.module.css:2]: We're still making this mistake within `ComponentTemplate`, might be a good idea to update it there.

**Review by lencodes — CHANGES_REQUESTED** [review]
Looks good @RalkeyOfficial!

Left some comments underneath.

**Review by remko48 — APPROVED** [review]

**Pull request merged**